Tests/PHPUnitTestCase.php

```diff
@@ -0,0 +1,5 @@
+<?php
```

Comment:
So it's better to raise requirements for symfony to `^2.8.18`. There's no value in supporting older versions